client/dashboard/emails/dataviews.tsx

```diff
@@ -72,15 +69,15 @@ export const emailFields: Field< Email >[] = [
 		label: __( 'Status' ),
 		render: ( { item }: { item: Email } ) => {
 			if ( item.status === 'active' ) {
-				return __( 'Active' );
+				return <span className="status status--active">{ __( 'Active' ) }</span>;
```

Comment:
We should use the existing `<Text>` component with `intent` prop: 

https://github.com/Automattic/wp-calypso/blob/a1faaf788dcf0d7b8658bfed4677b1bd9f100335/client/dashboard/components/text/index.tsx#L7

It has been implemented with the correct color stylings for the Hosting Dashboard.